src/App.js

```diff
@@ -1,21 +1,7 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Router from './Router';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
-}
+let App = () => <Router className="router"/>
```

Comment:
What does the router have a class name?

Reply:
I was using CSS to maintain width and height and wanted to ensure this remained consistent from the top.  I took off the class name, but may need to add it back later if we start having difficulty maintaining height to 100%.